ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -112,6 +122,8 @@ class FaultInjector
 
   // Map ow_lander::joint_t enum values to indices in JointState messages
   std::vector<unsigned int> m_joint_state_indices;
+
+  std::mt19937 m_randomGenerator; // Utilize a Mersenne Twister pesduo random generation
```

Comment:
should be m_random_generator

Reply:
Why is that? I thought we are using camelCase for member variables

Reply:
It seems to be the case .. I will update it

Reply:
snake_case - https://babelfish.arc.nasa.gov/confluence/display/OCEANWATERS/Coding+Style+and+Practices
This is why I wanted one IRG coding standard all along. There are too many too keep track of.

Reply:
addressed

Reply:
I have overlooked this from the beginning and thought the convention was m_CamelCase.  This is already pervasive in our code.  I would like to propose that we retain just the "m_" and not dictate the style that follows.  This would be sufficient to identify member variables.